partcad/src/partcad/assembly.py

```
@@ -28,15 +27,11 @@ def __init__(self, item, name=None, location=None):
 class Assembly(ShapeWithAi):
     path: typing.Optional[str] = None
 
-    def __init__(self, config={}):
-        super().__init__(config)
+    def __init__(self, project_name: str, config={}):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)